implementations/rust/ockam/ockam_node/src/node.rs

```diff
@@ -31,6 +31,7 @@ impl ockam_core::Worker for NullWorker {
 
 /// Start a node
 pub fn start_node() -> (Context, Executor) {
+    #[cfg(feature = "std")]
```

Comment:
IMHO let's move this to setup_tracing function and not expose those implementation details